django/db/models/sql/compiler.py

```diff
@@ -1748,6 +1748,7 @@ def as_sql(self):
                 for obj in self.query.objs
             ]
         else:
+            result.append("(%s)" % qn(opts.pk.column))
```

Comment:
This looks like an unrelated code change :thinking: @LilyFoote Can I revert it?

Reply:
I don't remember the reason I thought this was necessary. I think the new version is cleaner though, so worth landing in a different PR if not in this one.